libs/shared/src/components/CarrierBox.vue

```diff
@@ -17,14 +17,13 @@
 </template>
 
 <script lang="ts" setup>
-import {toRefs} from 'vue';
-import {type CarrierName} from '@myparcel/constants';
+import {resolveCarrierName} from '../utils';
+import {type CarrierIdentifier} from '../types';
 import {useCarrierRequest} from '../composables';
 import CarrierLogo from './CarrierLogo.vue';
 import Box from './Box.vue';
 
-const props = defineProps<{carrier: CarrierName}>();
-const propRefs = toRefs(props);
+const props = defineProps<{carrier: CarrierIdentifier}>();
 
-const {data} = useCarrierRequest(propRefs.carrier);
+const {data} = useCarrierRequest(resolveCarrierName(props.carrier));
```

Comment:
Dit is niet reactive, kijk naar hoe dit wordt gedaan in CarrierLogo.vue als voorbeeld

Reply:
Gedaan